src/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface Todo {
+  id: number;
+  title: string;
+}
+
+interface TodoListProps {
+  todos: Todo[];
+}
+
+const TodoList: React.FC<TodoListProps> = ({ todos }) => {
+  return (
+    <>
+      {todos.map((item) => {
+        return (
+          <pre key={item.id}>
+            {item.id} {item.title}
```

Comment:
There is no need to render id.

Reply:
react-jsx-dev-runtime.development.js:87 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `TodoList`. See https://reactjs.org/link/warning-keys for more information.
    at p
    at TodoList (http://localhost:3000/main.605c63a069248cc5a9bd.hot-update.js:26:5)
    at header
    at App (http://localhost:3000/static/js/bundle.js:32:76)